ddev/src/ddev/cli/dep.py

```diff
@@ -36,6 +36,7 @@
     'pydantic',
     # https://github.com/DataDog/integrations-core/pull/16080
     'lxml',
+    'oracledb',
```

Comment:
Could you include a little blurb saying why we ignore it and maybe some condition for when we can start upgrading it again?

Reply:
definitely! Let me know if that's clear enough

Reply:
nice 💪